buildkite/build_project.sh

```diff
@@ -264,6 +264,11 @@ case "$REPO_FULL_NAME" in
             make -C druntime -j2 -f posix.mak
         fi
         cd "$(basename "${REPO_FULL_NAME}")"&& make -f posix.mak clean && make -f posix.mak -j2 buildkite-test
```

Comment:
> What?!!! where?

This builds phobos as a dependency of `buildkite-test`

Reply:
I need Phobos to be built with this set to false:
https://github.com/dlang/phobos/blob/ba02646078ba45c9ad26ea348d0e9d7fad8f6a68/std/range/primitives.d#L2412

If there's already a CI that does this, I'll use it instead.

Reply:
Note, the `-version=NoAutodecodeStrings` directive does nothing for current Phobos. That is what I'm going to use to conditionally set that bool to false. But in order for things to test properly, I need a CI set up that will do it.

Reply:
I know, my  initial thought was to only build phobos with disabled autodecoding. But that doesn't work because the `betterC` tests were migrated from CircleCI to BuildKite ... guess we'll have to accept the double penalty.